### DISCUSSION TITLE
Custom docs path documentation not available in v2 docs

### DISCUSSION THREAD

**Discussion by skrmain** [facebook/docusaurus#3490] [General] [answered]
I recently build my notes site using docusaurus v2, `docs` help a lot, but I was restructuring the folder structure similar to how we have in v1. Like this

root-directory
├── docs
│   ├── doc1.md
└── website
    ├── blog
    │   └── 2017-10-24-new-version-1.0.0.md
    ├── package.json
    ├── pages
    ├── siteConfig.js
    ├── sidebars.json

Instead of the Default structure of v2. Basically moving the `docs` folder out of `website` folder. 

- To structure in that way I looked for options in the documentation but can't able to find something useful, as I think the `Advanced Guides/Presets` docs work is in progress.

**Comment by slorber** [comment, marked as answer]
Hi

You can put the docs folder wherever you want, there's a "path" option.

It's also mentioned in the migration guide: https://v2.docusaurus.io/docs/migrating-from-v1-to-v2/#docusaurusconfigjs

**Reply by skrmain** [reply]
Hi, @slorber 

I think we should have something mention about **Custom docs path** in `Advanced Guides/Presets`, also for the **custom blog path**.

If there are no other plan of adding these in docs. I would be happy to **contribute** it to the docs.

**Reply by slorber** [reply]
@skr571999 if you feel docs should be improved please feel free to submit a PR, any contribution is welcome and we certainly know v2 doc is far from perfect yet.

**Reply by softmarshmallow** [reply]
@slorber  The link is dead X(
